paddle/function/GemmConvOp.cpp

```diff
@@ -126,6 +126,11 @@ class GemmConvFunction : public ConvFunctionBase {
       inputData += inputChannels * inputHeight * inputWidth;
       outputData += outputChannels * outputHeight * outputWidth;
     }
+#ifdef PADDLE_MOBILE_INFERENCE
+    if (Device == DEVICE_TYPE_CPU) {
+      delete memory_;
```

Comment:
Need to use reset to release memory

Reply:
Done in https://github.com/PaddlePaddle/Paddle/pull/6666